source/deployment-options/offline-installation.rst

```diff
@@ -22,13 +22,24 @@ Prerequisites
 Download the packages and configuration files
 ---------------------------------------------
 
-#. Replace ``<deb|rpm>`` in the following command with your choice of package format and run it from a Linux system with Internet connection. This action executes a script that downloads all required files for the offline installation on x86_64 architectures.
+#. Run the following commands with your choice and execute them from a Linux system with Internet connection. This action executes a script that downloads all required files for the offline installation on x86_64 architectures.
```

Comment:
Suggestion for:
>`Run the following commands with your choice and execute them from a Linux system with Internet connection`

Change it to: 
> `Run the following commands with your choice from a Linux system with Internet connection`

Reply:
Done in https://github.com/wazuh/wazuh-documentation/pull/6686/commits/bae4dcc84a08e98181f4c81dcb125d83cc0c0596.